ui-library-filesharing-ui-components/api/functions/uploadFileToAzureBlobStore.ts

```diff
@@ -59,7 +59,7 @@ const httpTrigger = async function (req: HttpRequest, context: InvocationContext
     const fileContentBuffer = Buffer.from(fileContent);
     const size = fileContentBuffer.byteLength;
 
-    const uniqueFileName = `${v4()}-${file.name}`;
+    const uniqueFileName = `${v4()}-${CSS.escape(file.name)}`;
```

Comment:
Can JS escape function be used here instead?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape

Reply:
the encodeURI also doens't seem to work, causing error 500
<img width="939" alt="image" src="https://github.com/user-attachments/assets/30780738-c26f-4ca4-ac9e-f3fc4c612db5" />

Reply:
switch to using encodeUR